NOTICES.txt

```diff
@@ -34,6 +34,7 @@ Section 3: MIT
 >>> github.com/stretchr/testify - v1.7.0
 >>> gopkg.in/yaml.v3 - v3.0.0-20210107192922-496545a6307b
 >>> sigs.k8s.io/yaml - v1.3.0
+>>> github.com/natefinch/atomic - v1.0.1
```

Comment:
I think you previously mentioned the idea of creating a fork of this package (though I forget the reason). I'm wondering if it might be worth just adding an atomic file writer to a package in this repo. The implementation is on the small side. The atomic write implementations I've seen elsewhere write to a temp file then use `os.Rename` to finish the job.

Reply:
The reason for the fork was because the original package doesn't report if the temp file couldn't be deleted. Here's what I'm looking to add: https://github.com/natefinch/atomic/pull/17
It is indeed a small feature to implement, just a couple of go files. The only complicated part of it is the [Windows](https://github.com/natefinch/atomic/blob/master/file_windows.go) implementation, but I'm not sure we even need that.